typescript/packages/common-builder/src/built-in.ts

```diff
@@ -16,6 +16,28 @@ export function generateData<T>(
   return generateDataFactory(params);
 }
 
+export function fetchData<T>(
+  params: Value<{
+    url: string;
+    options?: RequestInit;
+    result?: T;
+    schema?: any;
+  }>
+): Value<{ pending: boolean; result: T; error: any }> {
```

Comment:
i don't know why it worked for generateData, but this should be CellProxy<> instead of Value<> for both to hopefully fix the .map problem

Reply:
I had made a mistake on my example:

```
-  const { result } = fetchData<{ result: Item[] }>({
+  const { result } = fetchData<Item[]>({
```

and rebasing on main with the export fix